english/is-this-a-dream/is-this-a-dream.md

```diff
@@ -7,6 +7,8 @@ No more eating at that strange ass pizzaria you promise your self...
 
 But then again .. none of this fabulous shit would have been yours had you not be a frequent visitor at
 Joe's hallucination pizza bar. You know you want it cause life is short and there is yet much to lord over
- .. so you pick up the phone and dial. "Hi I'd like to order a pizza ..."
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.